language-server/modules/langserver-commons/src/main/java/org/ballerinalang/langserver/commons/eventsync/PublisherKind.java

```diff
@@ -0,0 +1,27 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.ballerinalang.langserver.commons.eventsync;
+
+/**
+ * Represents the kinds of event publishers.
+ *
+ * @since 2201.1.1
+ */
+public enum PublisherKind {
+    PROJECT_UPDATE_EVENT_PUBLISHER
```

Comment:
Just a suggestion. WDYT changing this something like `TopicKind` class and `PROJECT_UPDATE`. ?
@IMS94 @gayalkuruppu thoughts?

Reply:
+1

Reply:
Discussed and named the class to `EventKind`